### PR TITLE
Update Event Router for 4.0

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/origin-aggregated-logging/pull/1522

https://jira.coreos.com/browse/OSDOCS-308

### CODE REVIEW

**Comment by mburke5678** [comment]
@jcantrill PTAL

**Comment by nhosoi** [comment]
@mburke5678, could you please advise?  I tried to see the entire page on the browser by accessing your original file as follows, but it only shows [1] and if I click modules/efk-logging-eventrouter-deploy.adoc, then it returns 404.  Could there be any way to open it?  Thanks!
https://github.com/mburke5678/openshift-docs/blob/logging-eventrouter/logging/efk-logging-eventrouter.adoc

[1]
```
Working with Event Router in {product-title}
The Event Router is a pod that forwards {product-title} events to Cluster Logging.
modules/efk-logging-eventrouter-deploy.adoc
```

**Comment by mburke5678** [comment]
@nhosoi There is a [link to a preview environment](https://github.com/openshift/openshift-docs/pull/13699#issuecomment-465758785) above. Try this link to see the topic in full:
https://logging-eventrouter--ocpdocs.netlify.com/openshift-enterprise/latest/logging/efk-logging-eventrouter.html

File: modules/efk-logging-eventrouter-deploy.adoc [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+apiVersion: v1
+metadata:
+  name: event-reader-binding
+subjects:
+- kind: ServiceAccount
+  name: cluster-logging-eventrouter <1>
+roleRef:
+  kind: ClusterRole
+  name: event-reader <2>
+  namespace: openshift-logging <3>
```
Comment (nhosoi) [inline modules/efk-logging-eventrouter-deploy.adoc]: Since it is a ClusterRoleBinding, we don't need namespace here?

Comment (mburke5678) [reply modules/efk-logging-eventrouter-deploy.adoc]: @nhosoi So, the template creates the service account, cluster role, and cluster role bindings? 
Did I misread @jcantrill README file:
_Elevated permissions are required to create the cluster level roles and bindings referenced in the following template_ 
That doesn't mean the template references existing CR and CRB?

Comment (nhosoi) [reply modules/efk-logging-eventrouter-deploy.adoc]: Hmmm, I thought you need to have an enough privilege to run "oc process -f <templatefile> | oc apply -f -"...  In my testing, I ran the command line with all-in-one <templatefile>, which made the eventrouter pod set up correctly... (note: without any prerequisites)
But yes, I think we'd better have an input from @jcantrill. ;)  Thanks!

Comment (mburke5678) [reply modules/efk-logging-eventrouter-deploy.adoc]: @nhosoi I removed the prereqs to create the service account, cluster role, and binding. Add ed a line about needing permission to update in `openshift-*` namespace, per @ewolinetz [comment in Slack](https://coreos.slack.com/archives/CB3HXM2QK/p1550874627165300).

Comment (nhosoi) [reply modules/efk-logging-eventrouter-deploy.adoc]: @mburke5678, thanks!
/lgtm

**Comment by mburke5678** [comment]
@anpingli PTAL

**Pull request merged**